packages/flame/lib/src/camera/viewfinder.dart

```diff
@@ -133,25 +137,30 @@ class Viewfinder extends Component
         'visibleGameSize cannot be negative: $value',
       );
       _visibleGameSize = value;
-      _initZoom();
+      _updateZoom();
     }
   }
 
   /// See [CameraComponent.visibleWorldRect].
   @internal
-  Rect get visibleWorldRect => _visibleRect ??= _computeVisibleRect();
-  Rect? _visibleRect;
-  Rect _computeVisibleRect() {
+  Rect get visibleWorldRect => visibleRect ??= computeVisibleRect();
```

Comment:
could this be a useful getter to expose to users?
sounds like something people might want to access for certain things

Reply:
It is exposed through the `CameraComponent`